backend/test/api/scores_test.clj

```diff
@@ -1,56 +1,59 @@
 (ns api.scores-test
   (:require [expectations.clojure.test :refer :all]
+            [clojure.test :as ctest]
+            [mocking :as mocking]
             [api.base :as api]
             [test-hand :as hand]
             [autoplay :as autoplay]
             [configs.hands :as hands]))
 
+(ctest/use-fixtures :each mocking/mock-persistence)
+
 (defexpect rows-won
   (expect
     #(= [3 1] 
         (:scores %))
     (autoplay/as-api
       (fn [i] (mod i 4))
       (fn [i] 0)))
-  (expect
-    #(= [2 2]
-        (:scores %))
-    (autoplay/as-api
-      (fn [i] (mod i 2))
-      (fn [i] (+ (mod i 2) 2)))))
+
+  (defexpect rows-tied
+    (expect
+      #(= [2 2]
+          (:scores %))
+      (autoplay/as-api
+        (fn [i] (mod i 2))
+        (fn [i] (+ (mod i 2) 2))))))
 
 (defexpect power-in-rows
-  ; Game begins with scores at 0
-  (expect
-    [[0 0] [0 0] [0 0] [0 0] [0 0]]
-    (let [game-state (api/create-game)
-          game-id (:game-id game-state)
-          player-id (:player-id game-state)
-          opponent-id (:player-id (api/add-player game-id))]
-      (:rows-power (api/get-game game-id player-id))))
-  (expect
-    [[(hand/power-of-nth 0) (hand/power-of-nth 1)]
-     [0 0] [0 0] [0 0] [0 0]]
-    (let [game-state (api/create-game)
-          game-id (:game-id game-state)
-          player-id (:player-id game-state)
-          opponent-id (:player-id (api/add-player game-id))]
+  (let [game-state (api/create-game)
+        game-id (:game-id game-state)
+        player-id (:player-id game-state)
+        opponent-id (:player-id (api/add-player game-id))]
+
+    ; Game begins with scores at 0
+    (expect
+      [[0 0] [0 0] [0 0] [0 0] [0 0]]
```

Comment:
Now we don't use `(repeat`? This is the decided guidelines for tests?

Reply:
I think yes, we should try to always have an `expect result` syntax, where it's the object expected, whatever its type or form (in some very few cases this might be terribly impossible or impractical, but they should be a minority)